lib/dom-parser.js

```diff
@@ -88,7 +88,7 @@ DOMHandler.prototype = {
 	    var len = attrs.length;
 	    appendElement(this, el);
 	    this.currentElement = el;
-	    
+
 		this.locator && position(this.locator,el)
```

Comment:
There seems to be some inconsistent indenting in the block of code that was updated here. I raised followup issue #29 to improve this (someday).